bigquery_etl/cli/backfill.py

```diff
@@ -220,32 +226,167 @@ def validate(
+                if not is_authenticated():
+                    click.echo(
+                        "Authentication to GCP required. Run `gcloud auth login` "
+                        "and check that the project is set correctly."
+                    )
+                    sys.exit(1)
+
+                client = bigquery.Client(project=project_id)
+                is_entry_processed = backfill_staging_table_exists(
+                    client, qualified_table_name
+                )
+
+                if processed == is_entry_processed:
+                    filtered_entries.append(entry)
 
-        entries_count = len(entries)
+            else:
+                filtered_entries.append(entry)
+
+        entries_count = len(filtered_entries)
 
         if entries_count:
             total_backfills_count += entries_count
 
-            project, dataset, table = extract_from_query_path(backfill_file)
-
+            project, dataset, table = qualified_table_name_matching(
+                qualified_table_name
+            )
             status_str = f" with {status} status" if status is not None else ""
             click.echo(
                 f"""{project}.{dataset}.{table} has {entries_count} backfill(s){status_str}:"""
             )
 
-            for entry in entries:
+            for entry in filtered_entries:
                 click.echo(str(entry))
 
     click.echo(f"\nThere are a total of {total_backfills_count} backfill(s).")
+
+
+@backfill.command(
+    help="""Process backfill.yaml with Drafting status that has not yet been processed.
+
+    Examples:
+
+    \b
+    # Process backfill entry for specific table
+    ./bqetl backfill process moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6
+
+    \b
+    # Process entries from all backfill.yaml files if table is not specified
+    ./bqetl backfill process
+
+    Use the `--project_id` option to change the project;
+    default project_id is `moz-fx-data-shared-prod`.
+    """
+)
+@click.argument("qualified_table_name", required=False)
+@sql_dir_option
+@project_id_option("moz-fx-data-shared-prod")
+@click.option(
+    "--force",
+    "--f",
+    help="Process the backfill entries even if a staging table exists",
+    default=False,
+    is_flag=True,
+)
+@click.option(
+    "--dry_run/--no_dry_run",
+    "--dry-run/--no-dry-run",
+    help="Dry run the backfill.  Note that staging table(s) will be deployed during dry run",
+)
+@click.pass_context
+def process(ctx, qualified_table_name, sql_dir, project_id, force, dry_run):
+    """Process backfill entry in backfill.yaml file(s)."""
+    click.echo("Backfill processing initiated....")
+
+    status = BackfillStatus.DRAFTING
+
+    backfills = get_qualifed_table_name_to_entries_map(
+        sql_dir, project_id, qualified_table_name, status.value
+    )
+
+    total_backfills_processed = 0
+
+    for qualifed_table_name, entries in backfills.items():
+        entry_to_be_processed = None
+
+        for entry in entries:
```

Comment:
Maybe error here if there's multiple entries just in case?

Reply:
error handling has been added to check if len(entries) > 1